waltz-tools/src/main/java/com/wepay/waltz/tools/cluster/ClusterCli.java

```diff
@@ -816,7 +873,8 @@ public String toString() {
                 PARTITION_ASSIGNMENT_ZK_SERVER_CONSISTENCY,
                 SERVER_STORAGE_CONNECTIVITY,
                 PARTITION_ASSIGNMENT_ZK_STORAGE_CONSISTENCY,
-                PARTITION_QUORUM_STATUS
+                PARTITION_QUORUM_STATUS,
+                STORAGE_RECOVERY_COMPLETE
```

Comment:
nit: naming convention. May be `Replica_recovery_status` or something instead?

Reply:
updated based on suggestion, PTAL😄